src/lib/mf.rs

```diff
@@ -825,6 +825,8 @@ impl Dist {
 
```

Comment:
Commenting here on the commit message: "...tests can specify terminal costs"?

Reply:
Ah, yes. I can fix that in a force push if that's OK?